### ISSUE TITLE
Microsoft.Azure.Cosmos.Table 2.0.0-preview needs to be deprecated

### ISSUE THREAD

**Issue by mario-guerra** [Azure/azure-cosmos-table-dotnet#78]
Track 2 SDKs for table storage have been released and we want to encourage customers to use the new packages. To that end, all existing track 1 packages need to be deprecated. 

[Microsoft.Azure.Cosmos.Table 2.0.0-preview](https://www.nuget.org/packages/Microsoft.Azure.Cosmos.Table/2.0.0-preview) needs to be deprecated as per our [guidelines for preview releases](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews).

**Comment by Edgar81** [comment]
Shouldn't all version of this package be deprecated? For instance 1.0.8 is not deprecated, although it depends on Microsoft.Azure.DocumentDB.Core - a package that is deprecated.

Thank you!

**Comment by mario-guerra** [comment]
Yes, we're working on deprecating a large number of older releases, to be announced at the end of the month.
